paste/translogger.py

```diff
@@ -29,9 +35,11 @@ def __init__(self, application,
                  logging_level=logging.INFO,
                  logger_name='wsgi',
                  setup_console_handler=True,
-                 set_logger_level=logging.DEBUG):
+                 set_logger_level=logging.DEBUG,
+                 time_format=None):
```

Comment:
why not assign the default right here?

```suggestion
                 time_format=default_time_format):
```

Reply:
Basically just to keep the same pattern used for `format` in line 28/34.

Which is perhaps not sufficient reason technically, but I tend to want to stick with existing patterns, especially in paste where everything is so weird and old and sometimes confusing.

Thus minimal changes.

Reply:
Makes sense, thank you.